DEVELOPMENT.md

```diff
@@ -195,7 +195,10 @@ for your `KO_DOCKER_REPO` if required. To be able to push images to
 ```shell
 gcloud auth configure-docker
 ```
-The [example GKE setup](#using-gke) in this guide grants service accounts permissions to push and pull GCR images in the same project.
+
+To be able to pull images from `gcr.io/<project>`, please follow the instructions [here](https://cloud.google.com/container-registry/docs/access-control#grant) to configure IAM policies for the services that will pull iamges from your GCR. A list of GCP default service accounts can be found [here](https://cloud.google.com/iam/docs/service-accounts#types)
```

Comment:
Can you refer back to https://github.com/tektoncd/pipeline/blob/main/DEVELOPMENT.md#using-gke that, that GKE creation would need the scope to be set, which is also actually allowing default compute SA to access GCP services. Here if you don't set the scope, you can simply set IAM for allowing default compute SA to have that access. And probably link to default compute SA

Reply:
I added a reference to https://github.com/tektoncd/pipeline/blob/main/DEVELOPMENT.md#using-gke and highlighted the scope setting. Also added a command snippet for the read access as alternative. @bendory